lib/charms/mongodb/v0/set_status.py

```diff
@@ -4,9 +4,9 @@
 # See LICENSE file for licensing details.
 import json
 import logging
-from typing import Tuple
+from typing import Optional, Tuple
 
-from charms.mongodb.v1.mongodb import MongoConfiguration, MongoDBConnection
+from charms.mongodb.v1.mongodb import MongoDBConfiguration, MongoDBConnection
```

Comment:
In a former PR, you unified the configuration which is now named `MongoConfiguration`.
```suggestion
from charms.mongodb.v1.mongodb import MongoConfiguration, MongoDBConnection
```